subo/command/compute_deploy_core.go

```diff
@@ -117,6 +121,18 @@ func ComputeDeployCoreCommand() *cobra.Command {
 
 				util.LogInfo("use `docker ps` and `docker-compose logs` to check deployment status")
 
+				proxy := localproxy.New("editor.suborbital.network")
+
+				go func() {
+					if err := proxy.Start(); err != nil {
+						log.Fatal(err)
+					}
+				}()
+
+				time.Sleep(time.Second * 2)
```

Comment:
I'm not a huge fan of this, but I do understand it's far less complex than using channels or something. Maybe add a comment?

Reply:
Sure thing